solution/serverless-test-files/serverless-backend.yml

```diff
@@ -0,0 +1,237 @@
+service: cmcs-eregs-site
+variablesResolutionMode: 20210326
+
+provider:
+  name: aws
+  runtime: python3.8
+  region: us-east-1
+  iam:
+    role: LambdaFunctionRole
+  lambdaHashingVersion: '20201221'
+  environment:
+    DB_NAME: eregstest 
+    DB_USER: eregstest1 
+    DB_PASSWORD: ${ssm:/eregulations/db/password}
+    DB_HOST: !GetAtt [RDSResource, Endpoint.Address]
+    DB_PORT: !GetAtt [RDSResource, Endpoint.Port]
+    #SURVEY_URL: ${ssm:/eregulations/survey_url}
+  vpc:
+    securityGroupIds:
+      - !Ref ServerlessSecurityGroup
+    subnetIds:
+      - subnet-021d6d373ec2d8c86
+      - subnet-0121c25b7306fe803
+  deploymentBucket:
+    blockPublicAccess: true
+package:
+  individually: false
+  exclude:
+    - node_modules/**
+    - nginx/**
+
+custom:
+#  stage: ${opt:stage, self:provider.stage}
+  wsgi:
+    app: handler.application
+    packRequirements: false
+  settings:
+    # static_url: ${cf:cmcs-eregs-static-assets-dev.StaticURL}/
```

Comment:
do we want to cleanup the commented out parts

Reply:
I left them in because I felt it was helpful to see what we do not include as a part of this but I'm fine with removing them

Reply:
removed